### ISSUE TITLE
Inline color picker

### ISSUE THREAD

**Issue by Ginger0913** [Simonwep/pickr#51]
#### Do you want to request a **feature** or report a **bug**? 
feature
#### What is the current behavior?
The current build is built for having a fixed poisition at all times, not allowing for the color picker to be inline without custom styling built around it.
#### What is the expected behavior?
Allow the developer configuring it to give a 'relative' position, which will bypass the fixed position and allow the user to have the color pickr act as a part of their form.
#### Your environment:
```
Pickr-version (see Pickr.version): Master (0.3.2 at time of writing)
Browser-version:  Firefox/Chrome
Operating-system:  Debian
```

**Comment by Ginger0913** [comment]
I created a pull request with the changes implemented to get this started. I belive that my linter might have messed up the indents a bit and went back to manually put most of it back to how it was by eye. I do apologize if this is an inconvenience. 
https://github.com/Simonwep/pickr/pull/52

**Comment by Simonwep** [comment]
No problem :) First of all thank's for this feature request, good point! I've add a comment to your PR (#52)!

**Comment by Simonwep** [comment]
An inline-option will be added.

**Comment by Simonwep** [comment]
Implemented in `0.4.9`
